File: net/management-api-v2/cm_api_v2_delete_workflow.cs
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    EnvironmentId = "KONTENT_AI_ENVIRONMENT_ID"
});

var identifier = Reference.ById(Guid.Parse("f9f28df0-9dec-4ee3-b087-c501e4b75347"));
// var identifier = Reference.ByCodename("my_workflow");

await client.DeleteWorkflowAsync(identifier);

File: ts/management-api-v2/cm_api_v2_delete_workflow.ts
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { ManagementClient } from '@kontent-ai/management-sdk';

const client = new ManagementClient({
  environmentId: 'KONTENT_AI_ENVIRONMENT_ID',
  apiKey: 'KONTENT_AI_MANAGEMENT_API_KEY'
});

const response = await client.deleteWorkflow()
  .byWorkflowId('f9f28df0-9dec-4ee3-b087-c501e4b75347')
  //.byWorkflowCodename('my-workflow')
  .toPromise();